module/rdpClientCon.c

```diff
@@ -1197,6 +1200,40 @@ rdpClientConInit(rdpPtr dev)
         g_sck_listen(dev->listen_sck);
         rdpClientConAddEnabledDevice(dev->pScreen, dev->listen_sck);
     }
+
+    ptext = getenv("XRDP_SESMAN_MAX_IDLE_TIME");
```

Comment:
I would prefer parsing for XRDP_SESMAN_MAX_IDLE_TIME to be added when it's actually supported. It's disappointing to find such code when figuring out why a feature doesn't work.